src/hyc_asd_mgr.py

```diff
@@ -34,10 +34,24 @@
 MODDED_FILE            = "/etc/aerospike/modded.conf"
 FILE_IN_USE            = None
 
-class ComponentStop(object):
-
-    def on_get(self, req, resp):
-        resp.status = HTTP_OK
+#
+# String Markers inserted in config file
+#
+CLEAN_DISKS_MARKER = "CLEAN_DISKS"
+DIRTY_DISKS_MARKER = "DIRTY_DISKS"
+MEMORY_PER_NS_MARKER = "MEMORY_PER_NS"
+MWC_MEMORY_MARKER = "MWC_MEMORY"
+PWQ_MEMORY_MARKER = "PWQ_MEMORY"
+
+config_high = { "max-write-cache" : 536870912,
```

Comment:
Can we have some formula which can make it generic. So that if compute config is changed from HA side, then we need not make changes in this code again